doc/specs/vulkan/chapters/initialization.txt

```diff
@@ -30,22 +30,30 @@ Loaders are encouraged to export function symbols for all other core
 Vulkan commands as well; if this is done, then applications that use only
 the core Vulkan commands have no need to use fname:vkGetInstanceProcAddr.
 
-Function pointers to commands that don't operate on a specific instance can:
-be obtained by using this command with pname:instance equal to `NULL`. The
-following commands can: be accessed this way:
-
-  * flink:vkEnumerateInstanceExtensionProperties
-  * flink:vkEnumerateInstanceLayerProperties
-  * flink:vkCreateInstance
-
-If pname:instance is a valid sname:VkInstance, function pointers to any
-commands that operate on pname:instance or a child of pname:instance can: be
-obtained. The returned function pointer must: only be called with a
-dispatchable object (the first parameter) that is a child of pname:instance.
-
-If pname:pName is not the name of a core Vulkan command, or is an
-extension command for any extension not supported by any available layer or
-implementation, then fname:vkGetInstanceProcAddr will return `NULL`.
+The table below defines the various use cases for fname:vkGetInstanceProcAddr
+and expected return value ("fp" is function pointer) for each case.
+
+.vkGetInstanceProcAddr behavior
+[width="80%",options="header",align="center"]
+|=====
+| pname:instance | pname:pName | return value
+| * | NULL | undefined
+| invalid instance | * | undefined
+| NULL | flink:vkEnumerateInstanceExtensionProperties | fp
+| NULL | flink:vkEnumerateInstanceLayerProperties | fp
+| NULL | flink:vkCreateInstance | fp
+| NULL | * (any pname:pName not covered above) | NULL
```

Comment:
Not sure I follow the nit picky comment: "there is a * "pName above" too".
I see an "invalid instance | * (pName)" case above, but that would only apply for invalid instance use and, by its nature, would have undefined behavior.